test/rubygems/test_gem_doctor.rb

```diff
@@ -154,6 +156,10 @@ def test_doctor_child_missing
   end
 
   def test_gem_repository_eh
+    unless (t = Dir.glob(File.join(@gemhome, "**/*.gemspec"))).empty?
```

Comment:
Why don't you use `util_clear_gemspecs` in this line?

Reply:
My mistake.  Early on, as I was running tests, I just had `util_clear_gems`.  As I saw more intermittent tests, I added `util_clear_gemspecs`.  I forgot to change the code here.  I'll fix, tomorrow (I'm -0600).